app/models/user.rb

```diff
@@ -48,6 +48,12 @@ def send_confirmation_instructions
     send_devise_notification(mailer, @raw_confirmation_token, opts)
   end
 
+  # send email to registered user if attempt is made to create account with registered email
+  def send_email_taken_notification
+    mailer = :email_taken
+    send_devise_notification(mailer)
```

Comment:
Thank you for the comment. Why not make it a one-liner

Reply:
Done that now